growthtools/ad-spend-calc
# Add disclaimer

Fixes [this To Do](https://3.basecamp.com/3097764/buckets/19037960/todos/3099726739)

### Testing notes

1. Visit ad spend calculator locally
1. See disclaimer under the Results card on the right hand side.
1. Should match [designs](https://projects.invisionapp.com/d/main/default/#/console/20628379/433566409/inspect).


### Other information
